import "rec/command/Command.proto";

package rec.command;

message CommandMapEntry {
  optional bytes key = 1;
  optional Command.Type command = 2;
};

message CommandMapProto {
  repeated CommandMapEntry entry = 1;
};